.circleci/config.yml

```diff
@@ -12,6 +12,7 @@ references:
       name: Decrypt secrets file
       command: |
         echo "${GIT_CRYPT_KEY}" | base64 -d > git-crypt.key
+        echo "${GIT_CRYPT_KEY}" | base64 -d > config/master.key
```

Comment:
👉 I think we confirm that this configuration shouldn't be necessary as the `RAILS_MASTER_KEY` env variable is being set instead, which (I believe) is the one that Rails uses.